views/layout.erb

```diff
@@ -1,17 +1,17 @@
 <!doctype html>
 <html>
   <head>
-    <title>The Landlord</title> 
+    <title>The Landlord</title>
     <link rel="stylesheet" type="text/css" href="/css/styles.css">
   </head>
   <body>
     <nav>
       <a href='/apartments'>List All apartments</a>
-      <a href='/apartments/3'>View sample apartment</a>
+      <a href='/'>View sample apartment</a>
       <a href='/apartments/new'>Create a new apartment</a>
-      <a href='/apartments/3/tenants'>List tenants for sample apartment</a>
-      <a href='/apartments/3/tenants/new'>Add tenant to apartment</a>
+      <a href='/'>List tenants for sample apartment</a>
+      <a href='/apartments/:id/tenants/new'>Add tenant to apartment</a>
```

Comment:
I know you don't have a tenants form yet, so this link can't work, but in general, you'd never use the `:id` type syntax inside a view, you'd want to put a **specific** id in this link, usually with ERB tags, and the specific apartment they want to add a tenant to (perhaps only on the show page for a specific apartment)